crates/router/src/configs/defaults.rs

```diff
@@ -1047,7 +1138,7 @@ impl Default for super::settings::RequiredFields {
                             (
                                 enums::Connector::Adyen,
                                 vec![RequiredFieldInfo {
-                                    required_field: "card_holder_name".to_string(),
+                                    required_field: "payment_method_data.bank_debit.ach_bank_debit.card_holder_name".to_string(),
```

Comment:
This needs to be generalized, cannot keep using `card_holder_name` for all the payment methods which are not related to cards.